site.yml

```diff
@@ -60,3 +60,30 @@
   - { role: swarm, run_as: worker }
   - { role: contiv_network, run_as: worker }
   - { role: contiv_storage, run_as: worker }
+
+# net-master hosts correspond to cluster machines that run the master
+# logic of the contiv net services
+- hosts: net-master
+  sudo: true
+  environment: env
+  roles:
+  - { role: base }
```

Comment:
I think I mis-understood your comment. Regardless, [devtest] has roles that I do not want to run (e.g. ceph). Additionally, base role will be run only once regardless of whether I add the host to [devtest] in inventory or not.